.github/workflows/deploy-changed-cf.yaml

```diff
@@ -155,7 +153,7 @@ jobs:
       max-parallel: 1
       matrix:
         environment-type: ${{ fromJson(needs.setup.outputs.matrix) }}
-        environment-name: [Biomage, HMS]
+        environment-name: [HMS]
         template: ${{fromJson(needs.get-modified-templates.outputs.files)}}
         exclude:
           - environment-name: HMS
```

Comment:
Line 158-160 can be removed as well I think? We only have 1 environment now right? HMS-production?

Reply:
they can, but I decided to keep them, just as an extra layer of protection